composer.json

```
@@ -3,5 +3,8 @@
     "description": "Customizations for PC",
     "type": "drupal-module",
     "license": "GPL-3.0-only",
-    "require": {}
+    "require": {
+        "discoverygarden/dgi_standard_oai": "^1.4",
+        "drupal/rest_oai_pmh": "dev-patch/incremental as 2.x-dev"
```

Just `^2` should be sufficient. The `as 2.x-dev` in the root project is an alias applied to the preceeding reference. See [the docs](https://getcomposer.org/doc/articles/aliases.md#require-inline-alias) for more info. Attempting to reference branches outside of the root composer project won't do anything.

```suggestion
        "drupal/rest_oai_pmh": "^2"
```